pkg/node/node.go

```diff
@@ -64,6 +65,13 @@ const (
 	maxTaintValueLength = 63
 )
 
+const (
+	// PodEvictReason is the event reason emitted for Pod evictions during node drain
+	PodEvictReason = "PodEviction"
+	// PodEvictMsg is the event message emitted for Pod evictions during node drain
+	PodEvictMsg = "Pod evicted due to node drain"
```

Comment:
@snay2 asked https://github.com/aws/aws-node-termination-handler/pull/631/files#r868346527

> Is this sufficient detail for the event? Do you need it to specify that node-termination-handler initiated the drain in response to a particular AWS event (Spot ITN, ASG event, etc.)?

I'm modifying the `PodEvictMsg` to `PodEvictMsgFmt` that will take a string parameter. Such string will be the `nodeName`, so the user will be able to relate a pod eviction to the node where the pod was running in. Then it will be possible to look for that node's events and see the AWS event that caused it.